board/opendingux/package/commander/commander.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+MODEL=$(sed -n 's/.*,\(.*\)ingenic.*/\1/p' /sys/firmware/devicetree/base/compatible)
```

Comment:
Note to self: Update this per https://github.com/OpenDingux/buildroot/commit/9562d2cb8ae17955f597025090783b369debd72a when rebasing

Reply:
Done